### ISSUE TITLE
Improving the report system

### ISSUE THREAD

**Issue by Eddy00707** [idena-network/idena-go#684]
For several consecutive validations, I'm seeing really bad 1-2-3-4 flips which don't end up reported only because they get exactly 50% of reports. Moreover, identities with the higher status (Verified or Human) generally report these flips, but the new users (Candidate or Newbie) usually do not. This makes sense logically since the older identities on average know more about the flips and the report system

Examples (both are 1-2-3-4 flips): 
https://scan.idena.io/flip/bafkreielecut4d7vhkbdmcqmeqkh5cyvv55vlf6a2ljku5ctpm2r5wbweq
https://scan.idena.io/flip/bafkreiechxpnoqlciaichh6g6d4jfyqq552kgmkibsqky3qknmntspbxua

My proposal:
- Decrease the report threshold from >50 to >=50, or
- give more weight to the reports from higher status identities, e.g. 1 report from Human might count as 1.5.

**Comment by leocornelius** [comment]
decreasing to a >=50 wont change much, and creates more problems than it solves. 50 is not a majority, and a 1% requirement change is only going to result in a tiny amount more 1234 flips being reported. Giving more reporting weight to higher status identities works well, perhaps also giving them more available reports.

**Comment by Eddy00707** [comment]
I agree with your point about 50% not being a majority. But changing it from >50 to >=50 is not a  tiny 1% change. Since, I believe, on average the flip has 6-8 people in the qualification committee, the threshold changes from 4/6 to 3/6 and from 5/8 to 4/8, which, in my examples, would be just enough to get those bad flips reported

**Comment by Toni-d-e-v** [comment]
The cool system will be human votes > newbie > candidates and that's if the flip is reported 51% = reported flip, 10 humans reports are = to 30 newbie ones, etc. As flip report attack on someone cant happen.
